src/styles/card.scss

```diff
@@ -0,0 +1,130 @@
+.container {
+  --column-count: 1;
+
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px;
+  padding-bottom: 16px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-top: 40px;
+    display: block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0;
+    margin-top: 4px;
```

Comment:
Similar to the `.card__title`, the `margin-top` on `.card__code` can be combined with the `margin: 0` reset into a single margin property.